crates/flox/src/commands/channel.rs

```diff
@@ -70,40 +83,99 @@ impl Search {
             })
             .sorted_by(|a, b| Ord::cmp(a, b));
 
-        if self.json {
-            let mut map = serde_json::Map::new();
-            for (channel, entry) in channels {
-                map.insert(
-                    entry.from.id.to_string(),
-                    json!({
-                        "type": channel.to_string(),
-                        "url": entry.to.to_string()
-                    }),
-                );
-            }
+        // Create `registry` parameter for `pkgdb`
+        let mut priority: Vec<String> = Vec::new();
+        let mut inputs = serde_json::Map::new();
+        for (_, entry) in channels {
```

Comment:
I think you could make the `filter_map` above just a `filter` since you aren't using `ChannelType`, and I'm not sure if the `sorted_by` is necessary either